src/Rings/MPolyQuo.jl

```diff
@@ -937,10 +960,8 @@ end
 
 #TODO: find a more descriptive, meaningful name
 function _kbase(Q::MPolyQuo)
-  I  = Q.I
-  GI = groebner_assure(I)
-  singular_assure(GI)
-  s = Singular.kbase(GI.S)
+    G = Q.I.gb[Q.ordering].gens.S
+  s = Singular.kbase(G)
```

Comment:
Indentation.

Reply:
Fixed, put the indention back to the old 2-spaces style. Should be unified now.